src/lib/button-toggle/README.md

```diff
@@ -1,139 +1 @@
-
-`md-button-toggle`s can be used with `ngModel` to dynamically create groups and update the value for
-a group.
-
-```html
-<md-button-toggle-group name="pies" [(ngModel)]="favoritePie">
-    <md-button-toggle *ngFor="let pie of pieOptions" [value]="pie">
-        {{pie}}
-    </md-button-toggle>
-</md-button-toggle-group>
-<p>Your favorite type of pie is: {{favoritePie}}</p>
-```
-
-### Disabled Button Toggle
-
-`md-button-toggle-group` and `md-button-toggle` can both be disabled by adding a `disabled`
-attribute to either one. Adding `disabled` to an exclusive group or a multiple group will disable
-the entire group. Adding `disabled` to a single toggle will disable that toggle.
-
-```html
-<md-button-toggle-group disabled>
-    <md-button-toggle value="one">One</md-button-toggle>
-    <md-button-toggle value="two">Two</md-button-toggle>
-    <md-button-toggle value="three">Three</md-button-toggle>
-</md-button-toggle-group>
-
-<md-button-toggle-group>
-    <md-button-toggle value="red" disabled>Red</md-button-toggle>
-    <md-button-toggle value="blue">Blue</md-button-toggle>
-</md-button-toggle-group>
-```
-
-Output:
-
-![Disabled Toggle Buttons Example](https://material.angularjs.org/material2_assets/button-toggle/disabled-toggles.png)
-
-## `<md-button-toggle>`
-
-### Bound Properties
-
-| Name | Type | Description |
-| --- | --- | --- |
-| `id` | string | The unique ID of the toggle. IDs are generated by default when not specified. |
-| `name` | string | Optional, defaults to parent group name if one exists for exclusive selection toggles, otherwise null. This is used to differentiate between different groups. |
-| `value` | any | Value of the toggle. Only used when in a group to determine which are selected. |
-| `checked` | boolean | Optional, default = `false`. Whether or not the toggle is checked. |
-| `disabled` | boolean | Optional, default = `false`. Whether or not the toggle is disabled |
-
-### Events
-
-| Name | Description |
-| --- | --- |
-| `change` | Emitted when the `checked` value is changed. |
-
-## `<md-button-toggle-group>`
-
-### Bound Properties
-
-| Name | Type | Description |
-| --- | --- | --- |
-| `name` | string | Optional, the name of the group. |
-| `disabled` | boolean | Optional, default = `false`. |
-| `value` | any | The current value for the group. Will be set to the value of the selected toggle or a list of values from the selected toggles. |
-| `selected` | `mdButtonToggle` | The current selected toggle or a list of the selected toggles. |
-| `vertical` | boolean | Whether the group should show the toggles vertically. Default is `false`. |
-
-### Attributes
-
-| Name | Type | Description |
-| --- | --- | --- |
-| `multiple` | boolean | Optional, default = `false`. Whether or not the group allows multiple selection. |
-
-### Events
-
-| Name | Description |
-| --- | --- |
-| `change` | Emitted when the `value` of the group changes. |
+https://material.angular.io/components/component/button-toggle
```

Comment:
Can you prefix each link with the phrase "Please see the official documentation at ", e.g.,
```
Please see the official documentation at https://material.angular.io/components/component/button-toggle
```

I also just realized that the path has "components/component". We need to fix that so that the url is just "https://material.angular.io/components/button-toggle". I've added that to the task list.